airbyte-integrations/connectors/source-shopify/pyproject.toml

```diff
@@ -17,7 +17,7 @@ include = "source_shopify"
 
 [tool.poetry.dependencies]
 python = "^3.9,<3.12"
-airbyte-cdk = "^0"
+airbyte-cdk = "0.81.4"
```

Comment:
This should not be changed.

Reply:
It's a change, I understand, but this looks to be better than what's happening with the open pinning. causing less issues.

Reply:
The actual code in use has not changed.

Reply:
If there are no functional changes, i don't think the source bump should take place either, we can safely merge this without source version bump, these changes than be picked up later on with the next functional change (bug fix or improvements). 

The cdk version pin will not help us later on when we can ant to be able to update the sources with the latest functionality causing to bump each and every source again.

Reply:
but I also updated all the descriptions

Reply:
do we plan to use that in UI?